frontend/utilities/helpers.ts

```diff
@@ -661,15 +661,21 @@ const DISK_ENCRYPTION_MESSAGES = {
   },
 };
 
-export const humanHostDiskEncryptionEnabled = (
-  platform?: string,
-  isDiskEncrypted = false
-): string => {
-  if (platform !== "windows" && platform !== "darwin") {
+export const getHostDiskEncryptionTooltipMessage = (
+  platform: "darwin" | "windows" | "chrome", // TODO: improve this type
+  diskEncryptionEnabled = false
+) => {
+  if (platform === "chrome") {
+    return "Fleet does not check for disk encryption on Chromebooks, as they are encrypted by default.";
+  }
+
+  // is this logic necessary? preserved from previous implementation
+  if (!["windows", "darwin"].includes(platform)) {
```

Comment:
Here's [the specs](https://github.com/fleetdm/fleet/issues/7440) for the original tooltip messages

Reply:
Can you please explain what you're getting at?

Reply:
This logic is necessary because it shows for only windows and darwin, we never show it for linux. Idk if you wrote line 672 or not as something to be addressed